src/main/java/com/spotify/futures/CompletableFutures.java

```diff
@@ -65,6 +66,30 @@ private CompletableFutures() {
    */
   public static <T> CompletableFuture<List<T>> allAsList(
       List<? extends CompletionStage<? extends T>> stages) {
+    return allAsList(stages, null);
```

Comment:
You could pass in something similar to `NoFastFail.INSTANCE` instead of null

Reply:
:+1: will do. 
- [ ] Skip fail fast setup by comparing an instance instead of null.